src/basic/terminal-util.h

```diff
@@ -111,6 +111,10 @@ typedef enum ColorMode {
         /* Only 256 colors. */
         COLOR_256 = 256,
 
+        /* COLORTERM default value truecolor for terminator, 16,777,216 colors. 24-bit color Value. */
+        /* Other Possible value gnome-terminal */
```

Comment:
I am not native, but the comment sounds strange for me.

At least, please merge the two lines, that is, drop the slashes at the end of the first line, and the beginning of the second line.

Reply:
Updated comment in sync with previous comment for other values.